<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>802</width>
    <height>585</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>802</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 203, 204);</string>
  </property>
  <widget class="QLabel" name="lbl_Date">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);



</string>
   </property>
   <property name="text">
    <string>Bienvenue sur BabyID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Listes">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(77, 77, 77);
font: 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Consulter le registre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_NewDeclaration">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>480</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);
font: 14pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Nouvelle déclaration</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>190</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Nombre d'enregistrements aujourd'hui :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_NbreNaissancesJour">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>190</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ce jour</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>250</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Nombre d'enregistrements ce mois :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_NbrNaissancesMois">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Ce mois</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>320</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Nombre d'enregistrements cette année :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_NbrNaissancesYear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>310</y>
     <width>251</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Times New Roman&quot;;
background-color: rgba(255, 255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Cette annee</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
